RIOT-OS/RIOT
# doc/doxygen: remove default value for search form

### Contribution description

The search input field on doc.riot-os-org is prefilled with `value="Search"`, instead of only using the HTML `placeholder` attribute. I think this was not the case until some recent doxygen version bump, but I am not sure since the corresponding JS-code has been part of the repo for seven years.


### Testing procedure

Open https://doc.riot-os.org on Firefox or Chrome, see that the search field is prefilled.

Compare to the CI generated doxygen, which should only use "Search" as placeholder value: https://ci.riot-os.org/results/534d783017ac45f0a2d217c67b876c9f/doc-preview/


### Issues/PRs references

Pinging @miri64 as the original author of the js-code.
